api/soap/mantisconnect.wsdl

```diff
@@ -269,6 +272,7 @@
  <xsd:complexType name="FilterSearchData">
   <xsd:all>
    <xsd:element name="project_id" type="tns:IntegerArray" minOccurs="0"/>
+   <xsd:element name="alias" type="xsd:string" minOccurs="0"/>
```

Comment:
If the report issue API is going to handle all the internals of the bug reporting, then I don't know we would need to expose the alias in filter API.